src/main/java/nl/tudelft/ewi/devhub/server/database/controllers/AssignedTAs.java

```diff
@@ -0,0 +1,51 @@
+package nl.tudelft.ewi.devhub.server.database.controllers;
+
+import com.google.inject.Inject;
+import com.mysema.query.jpa.JPASubQuery;
+import nl.tudelft.ewi.devhub.server.database.entities.*;
+
+import javax.persistence.EntityManager;
+import javax.transaction.Transactional;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static com.mysema.query.group.GroupBy.groupBy;
+import static com.mysema.query.group.GroupBy.list;
+import static java.util.Comparator.comparing;
+import static nl.tudelft.ewi.devhub.server.database.entities.QAssignedTA.assignedTA;
+import static nl.tudelft.ewi.devhub.server.database.entities.QDelivery.delivery;
+
+/**
+ * Created by sayra on 07/06/2017.
+ */
+public class AssignedTAs extends Controller<AssignedTA> {
```

Comment:
Even though the `AssignedTAs` class is probably hit in the end to end test suite, it still may be an idea to write an integration test that checks whether the relatively complex query defined in `AssignedTAs#getLastDerliveries(Assignment, User)` actually works (or at least executes). See also how the other controllers/ Data Access Objects are tested at https://github.com/devhub-tud/devhub/tree/master/src/test/java/nl/tudelft/ewi/devhub/server/database/controllers .